### ISSUE TITLE
Show ip address in machine list

### ISSUE THREAD

**Issue by avanzzzi** [avanzzzi/hypy#22]
Show ip address in machine list

**Comment by ArthurGorgonio** [comment]
How about something like a flag in the `list/ls` command to show the machine ip or better we always show this one?

I think something like a flag in the list command and/or configuration file to make it easy to set.What do you think?

**Comment by avanzzzi** [comment]
Yeah, a flag in the list command makes sense, although my first thought was to show the IP Address by default.

I'm not sure but maybe a second Powershell command will be needed to get the IP so a flag works better.
